book/src/prerequisites.md

```diff
@@ -31,7 +31,7 @@ cargo run
 
 ## Cargo + Meson
 
-Cargo is *nearly* enough, but it is not well suited for handling resources such as icons or UI files.
+Cargo is *nearly* enough, but it is not well suited for handling resources such as icons or `xml` files.
```

Comment:
xml files as in what? html files?

Reply:
Ui files, gressource files and gschema files

Reply:
well, they should be mentioned explicitly then :)

Reply:
Mmmm, I wanted to limit jargon at the beginning. But I have no strong feelings there.

Reply:
UI files seems less jargon than a generic XML. Maybe make it "UI definition files"?

Reply:
Fixed